spec/fixtures/api/crash.html

```diff
@@ -20,7 +20,7 @@
     })
 
     if (process.platform === 'win32') {
-      ipcRenderer.sendSync('crash-service-pid', crashReporter.crashServicePid)
+      ipcRenderer.sendSync('crash-service-pid', crashReporter._crashServiceProcess.pid)
```

Comment:
why are you changing this?

Reply:
`_crashServiceProcess` is not defined

Reply:
https://github.com/electron/electron/commit/508551b81b3e76998b38e6d56aaa0de47e488664#diff-ba522aa2d4f3759eca6524511f8b9442